sel4-support.h

```diff
@@ -0,0 +1,172 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
```

Comment:
I realized we need these definitions for other guest Linux kernel modules also. We should create a separate Yocto package that contains this header.

Reply:
Moved this part to different repo and made new recipe 4 it.